templates/admin-classes.html

```diff
@@ -8,23 +8,25 @@ <h1>Classes overview</h1>
     </div>
 </div>
 <h4 class="text-center">Total amount of shown classes: {{ classes|length }}</h4>
-<div class="program w-1/2 mx-auto border-solid border-2 border-orange-400 rounded p-4">
+<div class="program w-3/4 mx-auto border-solid border-2 border-orange-400 rounded p-4">
   <table class="table-auto mb-4 w-full text-sm">
     <thead>
       <tr class="font-bold text-left">
           <td id="classname_header">Class name</td>
           <td id="teacher_header">Teacher</td>
           <td id="students_header">Students</td>
+          <td id="last_used_header">Last used</td>
           <td id="statistics_header">Statistics</td>
       </tr>
     </thead>
     <tbody>
       {% for class in classes %}
           <tr>
-            <td class="classname_header">{{class.name}}</td>
-            <td class="teacher_header">{{class.teacher}}</td>
-            <td class="students_header">{{class.students|length}}</td>
-              <td id="statistics_header"><a href="/stats/class/{{class.id}}" target="_blank">Click here</a></td>
+            <td class="classname_cell">{{class.name}}</td>
```

Comment:
We're renaming `_header` to `_cell` here, is this intentional? Mostly asking to be sure!

Reply:
Yes! We use a similar structure on our other tables where we support dynamic showing / hiding of column. We don't have this feature (yet) on the classes overview, but might be useful in the future. By using the same naming structure we unify the code and, the biggest advantage, can call the same function for the functionality, this one:

```
$(document).ready(function() {
        $('.attribute').change(function() {
            const attribute = $(this).attr('id');
            if(!this.checked) {
                $('#' + attribute + '_header').hide();
                $('.' + attribute + '_cell').hide();
            } else {
                $('#' + attribute + '_header').show();
                $('.' + attribute + '_cell').show();
            }
        });
});
```